### ISSUE TITLE
v12 Conforming

### ISSUE THREAD

**Issue by spergmoment** [spergmoment/FederalBot#29]
Recently, discord.js v12 got pushed to stable.
This version introduces so many changes it's actually fucking insane.

I won't list them here, but the general stuff that I need to consider is:
- NEW EVENTS!
  * MEMBER_MOVE
  * MEMBER_DISCONNECT
  * BOT_ADD
  * MESSAGE_BULK_DELETE
  * MESSAGE_PIN
  * MESSAGE_UNPIN
  * INTEGRATION_CREATE
  * INTEGRATION_UPDATE
  * INTEGRATION_DELETE
- I won't need to add any besides bulk delete and pin/unpin.
- You must access caches now, as managers replaced storages/collections
- Add role and remove role have been replaced with directly managing a member's roles
- Some user properties changed to methods
- # RICHEMBEDS HAVE BEEN REPLACED WITH MESSAGEMBEDS!
- String concatenation is no longer supported, conform any remains to ES6 string lits
- Pings have changed to being managed by a guild shard manager
- Various small changes no one gives a fuck about

Basically, I have to change a SHIT ton.

**Comment by spergmoment** [comment]
Finished conforming!
